meinberlin/apps/budgeting/predicates.py

```diff
@@ -0,0 +1,10 @@
+import rules
+
+from adhocracy4.phases.predicates import has_feature_active
+
+
+@rules.predicate
+def phase_allows_vote(user, item):
+    if item:
+        return has_feature_active(item.module, item.__class__, 'vote')
+    return False
```

Comment:
Maybe move this to A4 to all other phase predicates? But on the other hand we only use that here .. @fuzzylogic2000 ?

Reply:
Yes, it should be moved, I think.